### ISSUE TITLE
[3.0.0-rc4] [regression] Sourcemap not found with expo-updates

### ISSUE THREAD

**Issue by RodolfoGS** [expo/sentry-expo#149]
### Summary of Issue 

Seems that this issue https://github.com/expo/sentry-expo/issues/134 is regression with the latest versions of expo, expo-updates and sentry-expo

I'm testing the `3.0.0-rc4`. If you install an update via `expo-updates`, Sentry can't found the sourcemaps.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

```
  Expo CLI 3.27.13 environment info:
    System:
      OS: macOS 10.15.5
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 14.13.0 - /usr/local/bin/node
      Yarn: 1.19.1 - /usr/local/bin/yarn
      npm: 6.14.8 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.9.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.0, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
      Android SDK:
        Android NDK: 21.3.6528147
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
      Xcode: 12.0/12A7209 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~39.0.2 => 39.0.3
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: ~0.63.3 => 0.63.3
      react-native-web: ~0.13.12 => 0.13.14
    npmGlobalPackages:
      expo-cli: 3.27.13
    Expo Workflow: bare
```

### Reproducible Demo

Repo: https://github.com/RodolfoGS/sentry-test

1. `git clone git@github.com:RodolfoGS/sentry-test.git`
2. `cd sentry-test`
3. `yarn install`
4. Configure the following files with your Sentry values (org, project, token and dsn)
```
 - android/sentry.properties
 - app.json
 - ios/sentry.properties
 - App.js
```
5. Build APK as release
6. Install in your device, run and tap on `Crash Now` button
7. The crash appears perfectly on Sentry, with their sourcemaps 👍
8. `expo publish`
9. Tap on `Update Now` button
10. Tap on `Crash Now` button
11. The crash appears on Sentry without their sourcemaps 👎

I'm experimenting this issue on Android and iOS

**Comment by zibs** [comment]
So I'm also experiencing this issue @cruzach, which we're also [tracking here on Sentry's repo](https://github.com/getsentry/sentry-react-native/issues/1116).

Using:
```
"expo-updates": "^0.3.5",
"react-native": "0.63.3",
"sentry-expo": "^3.0.2",
"@sentry/react-native": "^1.9.0",
```

The errors show properly when it's the first build from Xcode in TestFlight, but any OTA updates afterwards are not showing properly.

I'm using the `postExport` hook to upload the source-maps to sentry, which seems to work (although I've had their support mention that the `sourceMappingURL` looks funny on the bundles that get uploaded.

A bundle uploaded by Xcode has `//# sourceMappingURL=main.jsbundle.map` as its very last line (which is what Sentry expects supposedly), while a bundle uploaded by this hook has this:
```
//# sourceMappingURL=http://127.0.0.1:19001/index.map?dev=false&minify=true&hot=false&platform=ios
//# sourceURL=http://127.0.0.1:19001/index.bundle?dev=false&minify=true&hot=false&platform=ios
```
which could potentially be causing issues. However, even if I re-upload these source-maps, having edited them to be more like the Xcode uploaded versions, it still doesn't work.

I've had support confirm that the dist/release values all seem to align and look good to them on both the OTA updates and the original releases.

Any insight or ideas are appreciated!

**Comment by cruzach** [comment]
waiting for https://github.com/getsentry/sentry-javascript/pull/3070 then will re publish sentry-expo with the fix 👍

**Comment by cruzach** [comment]
@zibs and @RodolfoGS , fix is released in `sentry-expo@3.0.4`
